test/Microsoft.IdentityModel.Tokens.Tests/JsonWebKeySetTests.cs

```diff
@@ -23,7 +23,14 @@ public void Constructors(JsonWebKeySetTheoryData theoryData)
             {
                 var jsonWebKeys = new JsonWebKeySet(theoryData.Json);
                 var keys = jsonWebKeys.GetSigningKeys();
+                var originalString = jsonWebKeys.JsonData;
                 theoryData.ExpectedException.ProcessNoException(context);
+
+                if (theoryData.Json != null)
+                    IdentityComparer.AreStringsEqual(originalString, theoryData.Json, context);
+                else
+                    IdentityComparer.AreStringsEqual(originalString, string.Empty, context);
+
```

Comment:
I think `JsonWebKeySetTheoryData.JsonData` should be set to `string.Empty` by default, since that's the default test case we're testing for, and then this code wouldn't be needed.